src/include/planner/plan_util.h

```diff
@@ -90,6 +90,12 @@ class PlanUtil {
         table_ids.insert(insert_node->GetTable()->GetOid());
         break;
       }
+      /*case PlanNodeType::POPULATE_INDEX: {
```

Comment:
The latest commit already has it uncommented. I think if it is okay with you, this is good to go.